pkg/api/api.go

```diff
@@ -186,8 +186,9 @@ type IssueCodeRequest struct {
 	TestType    string `json:"testType"`
 	// Offset in minutes of the user's timezone. Positive, negative, 0, or omitted
 	// (using the default of 0) are all valid. 0 is considered to be UTC.
-	TZOffset float32 `json:"tzOffset"`
-	Phone    string  `json:"phone"`
+	TZOffset         float32 `json:"tzOffset"`
+	Phone            string  `json:"phone"`
+	SMSTemplateLabel string  `json:"SMSTemplateLabel"`
```

Comment:
This should be `smsTemplateLabel` (I don't make the rules)

Reply:
yea that's consistent with tzOffset. Weird though.